src/tools/msvc.jam

```diff
@@ -689,15 +689,15 @@ rule compile.c ( targets + : sources * : properties * )
 {
     set-setup-command $(targets) : $(properties) ;
     get-rspline $(targets) : -TC CFLAGS  ;
-    compile-c-c++ $(<) : $(>) [ on $(<) return $(PCH_FILE) ] [ on $(<) return $(PCH_HEADER) ] ;
+    compile-c-c++ $(<) : $(>) ;
```

Comment:
I'm actually not sure why it was done like this, maybe I miss something, like it's need to run header scanning or something else?